seqr/utils/elasticsearch/es_search.py

```diff
@@ -807,7 +807,8 @@ def _parse_hit(self, raw_hit):
         })
         return result
 
-    def _parse_genotypes(self, raw_hit, hit, index_family_samples, is_sv):
+    def _parse_genotypes(self, raw_hit, hit, index_family_samples, data_type):
+        is_sv = data_type == Sample.DATASET_TYPE_SV_CALLS
```

Comment:
this doesn't appear to be used

Reply:
We used it twice. But only one is left after we changed `GENOTYPE_FIELDS_CONFIG` to `GENOTYPE_FIELDS[data_type]`. I will remove `is_sv` by using its value directly.